### DISCUSSION TITLE
C# How get InventoryController?

### DISCUSSION THREAD

**Discussion by Eugeny335** [praydog/REFramework#1116] [General Questions]
How can I get a player's Inventory Controller using the c# API?Found only for lua "player_HeadUpdater:get_InventoryController()", but in c sharp there is no "get_InventoryController()" function in HeadUpdater

**Comment by praydog** [comment]
![dfgfsd](https://github.com/user-attachments/assets/49c8dd83-485a-4ac2-b93b-03dfef94ef70)
Yes there is. It's a property, not a standard method.

**Comment by Eugeny335** [comment]
> ![dfgfsd](https://github.com/user-attachments/assets/49c8dd83-485a-4ac2-b93b-03dfef94ef70)
> 
> Yes there is. It's a property, not a standard method.

And there is no example, how can I get this?
